c/c11_support.c

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
Please separate out the files copied from other sources into its own commit. This can make tracking down origin and copyright holder substantially easier. Include something in the commit message about were it came from:

"<Summary line>

These Apache-licensed files were copied as-is from <repo_url> at commit <commit>.

<Explanation of why>"